app/views/auctions/_auction_status.html.erb

```diff
@@ -1,4 +1,4 @@
-<span class="bid-amount"><%= @view_model.auction_current_bid_amount_as_currency %></span>
+<span class="bid-amount"><%= number_to_currency(@view_model.highlighted_bid_amount) %></span>
```

Comment:
Number helpers are already in lots of these internal classes. So, I would move this helper call out to the class responsible.